checkov/bitbucket_pipelines/runner.py

```diff
@@ -0,0 +1,135 @@
+import os
+
+from checkov.bitbucket_pipelines.checks.registry import registry
+from checkov.common.images.image_referencer import ImageReferencer, Image
+from checkov.common.output.report import CheckType
+from checkov.yaml_doc.runner import Runner as YamlRunner
+
+
+class Runner(YamlRunner, ImageReferencer):
+    check_type = CheckType.BITBUCKET_PIPELINES
+
+    def __init__(self):
+        super().__init__()
+
+    def require_external_checks(self):
+        return False
+
+    def import_registry(self):
+        return registry
+
+    def _parse_file(self, f):
+        if self.is_workflow_file(f):
+            return super()._parse_file(f)
+
+    def is_workflow_file(self, file_path):
+        """
+        :return: True if the file mentioned is in a is named bitbucket-pipelines.yml. Otherwise: False
+        """
+        abspath = os.path.abspath(file_path)
+        return abspath.endswith(("bitbucket-pipelines.yml", "bitbucket-pipelines.yaml"))
```

Comment:
```suggestion
        return file_path.endswith(("bitbucket-pipelines.yml", "bitbucket-pipelines.yaml"))
```
no need to evaluate the `abspath`, because you are not using it